packages/radio/src/index.ts

```diff
@@ -1,9 +1,33 @@
-import { FrameworkConfiguration, PLATFORM } from 'aurelia-framework';
+import { FrameworkConfiguration, PLATFORM, bindingMode, ObserverLocator, CheckedObserver } from 'aurelia-framework';
+import { AureliaUX } from '@aurelia-ux/core';
 
 export { UxRadioTheme } from './ux-radio-theme';
 
 export function configure(config: FrameworkConfiguration) {
+  config.container.get(AureliaUX).registerUxElementConfig(uxCheckBoxConfig);
   config.globalResources([
     PLATFORM.moduleName('@aurelia-ux/radio/ux-radio')
   ]);
 }
+
+const uxCheckBoxConfig = {
```

Comment:
We need to be sure to extend this configuration to the [`@aurelia-ux/components`](https://github.com/aurelia/ux/tree/master/packages/components) package.

Reply:
Shouldn't this be `uxRadioConfig` rather than `uxCheckBoxConfig`?